pkgs/servers/ursadb/default.nix

```diff
@@ -1,38 +1,46 @@
-{ lib, stdenv, fetchFromGitHub, cmake, zeromq, cppzmq }:
+{ lib, stdenv, fetchFromGitHub, cmake, zeromq, cppzmq, spdlog }:
 
 stdenv.mkDerivation rec {
   pname = "ursadb";
-  version = "1.2.0";
+  version = "1.5.0";
 
   src = fetchFromGitHub {
     owner = "CERT-Polska";
     repo = "ursadb";
     rev = "v${version}";
-    hash = "sha256-/EK1CKJ0IR7fkKSpQkONbWcz6uhUoAwK430ljNYsV5U=";
+    hash = "sha256-JssL1Yd+2CPy9fDPJnstiRucdy6TyUdb0Q6xa160CLM=";
+    fetchSubmodules = true;
   };
 
   postPatch = ''
     substituteInPlace CMakeLists.txt \
-      --replace \
-        "add_executable(ursadb_test Tests.cpp)" "" \
-      --replace \
-        "target_link_libraries(ursadb_test ursa)" ""
+      --replace "add_executable(ursadb_test src/Tests.cpp)" "" \
+      --replace "target_link_libraries(ursadb_test ursa)" "" \
+      --replace "target_enable_ipo(ursadb_test)" "" \
+      --replace "target_clangformat_setup(ursadb_test)" "" \
+      --replace 'target_include_directories(ursadb_test PUBLIC ${"$"}{CMAKE_SOURCE_DIR})' "" \
+      --replace "ursadb_test" ""
   '';
 
   installPhase = ''
     mkdir -p $out/bin
+
```

Comment:
I think this whole override could be dropped as the stdenv defaults would do the same thing

Reply:
Looks like you're right - I've removed it altogether,